apps/web/lib/video/[uid]/getServerSideProps.ts

```diff
@@ -134,6 +139,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
           : bookingObj.user,
       },
       hasTeamPlan: !!hasTeamPlan,
+      calVideoLogo,
```

Comment:
I'd rather call a cache-able url directly from an img tag like with the  logo API endpoint. This will still get us huge html payloads.